extensions/security/audittrail/applib/src/main/java/org/apache/causeway/extensions/audittrail/applib/dom/AuditTrailEntryRepository.java

```diff
@@ -38,6 +39,10 @@ public interface AuditTrailEntryRepository {
 
     AuditTrailEntry createFor(final EntityPropertyChange change);
 
+    default Can<AuditTrailEntry> createForBulk(final Can<EntityPropertyChange> entityPropertyChanges) {
+        return Can.empty();
```

Comment:
I think that the default implementation of this should iterate over the provided `Can<..>` and call the original `createFor(...)`, mapping to the returned `Can<>`.

Reply:
Fixed